src/apis/servicing_api.ts

```diff
@@ -170,6 +180,30 @@ export class ServicingAPI {
         return debtRegistry.get.callAsync(issuanceHash);
     }
 
+    public async getRepaymentScheduleAsync(issuanceHash: string): Promise<Array<number>> {
+        this.assert.schema.bytes32("issuanceHash", issuanceHash);
+
+        const debtRegistryEntry = await this.getDebtRegistryEntry(issuanceHash);
+
+        const { termsContract } = debtRegistryEntry;
+
+        /*
+         *  TODO: Figure out the which adapter to use - e.g. SimpleInterestLoanAdapter.
+         *  If the simple interest contract has been loaded at all, then it will be
+         *  necessarily in the contract APIs cache. The address in the debt order and debt entry
+         *  must then be the same as the address in the cache.
+         *
+         *  For now, assume that it's the SimpleInterestLoanAdapter.
+         */
+        const adapter = new SimpleInterestLoanAdapter(this.web3, this.contracts);
+
+        if (!adapter) {
```

Comment:
Though we should definitely throw an error here, I don't think this error will ever be triggered by this -- i.e. `adapter` will always be defined.